eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -20,6 +20,16 @@ jobs:
           $versioningProperties = "/p:OfficialBuildId=$(OfficialBuildId) /p:SkipDevBuildNumber=$skipDevBuildNumber";
           echo "##vso[task.setvariable variable=VersioningProperties]$versioningProperties"
         displayName: "Setup .NET specific versioning properties"
+      - task: UsePythonVersion@0
```

Comment:
We have been trying to have unmutated files in our packages so I would prefer we not have moved this. However with @KarishmaGhiya link work this will end up being completely removed anyway so I'm fine with it being here until then.